lib/elixir/lib/code.ex

```diff
@@ -104,6 +104,10 @@ defmodule Code do
   The `binding` argument is a keyword list of variable bindings.
   The `opts` argument is a keyword list of environment options.
 
+  **Warning**: `string` is assumed to be fully trusted.  If you receive strings
+  (for example, over the network), passing them into this function can execute
+  arbitrary code and compromise your machine.
```

Comment:
> maybe "executed without any kind of sandbox" or "executed with full privileges"

What about "executed with the same privileges of the Erlang VM"?

> Also, eval_string instead of eval_string/3 because it applies to /1 and /2 as well?

We always refer to the highest arity when a function is present in different arities because of default arguments. :)